src/main/java/seedu/address/model/person/Name.java

```diff
@@ -9,8 +9,9 @@
  */
 public class Name {
 
-    public static final String MESSAGE_CONSTRAINTS = "Names should only contain alphanumeric characters and spaces, "
-            + "and it should include given name and surname!";
+    public static final String MESSAGE_CONSTRAINTS = "Name must consist only of letters.\n"
+            + "Each name part is separated by exactly one space.\n"
```

Comment:
Would `Name should only contain alphabet characters and spaces` be a better description? Also not very sure if adding the `Each name part is separated by exactly one space` is allowed